npm_and_yarn/spec/dependabot/npm_and_yarn/registry_helper_spec.rb

```diff
@@ -0,0 +1,154 @@
+# typed: false
+# frozen_string_literal: true
+
+require "dependabot/dependency_file"
+require "dependabot/npm_and_yarn/registry_helper"
+require "spec_helper"
+
+RSpec.describe Dependabot::NpmAndYarn::RegistryHelper do
+  let(:npmrc_file) do
+    Dependabot::DependencyFile.new(
+      name: ".npmrc",
+      content: <<~NPMRC
+        registry=https://custom-registry.com/
+        _authToken=custom-token
+      NPMRC
+    )
+  end
+
+  let(:npmrc_without_token_file) do
+    Dependabot::DependencyFile.new(
+      name: ".npmrc",
+      content: <<~NPMRC
+        registry=https://custom-registry.com/
+      NPMRC
+    )
+  end
+
+  let(:empty_npmrc_file) do
+    Dependabot::DependencyFile.new(name: ".npmrc", content: "")
+  end
+
+  let(:yarnrc_file) do
+    Dependabot::DependencyFile.new(
+      name: ".yarnrc",
+      content: <<~YARNRC
```

Comment:
`.yarnrc` is usually space delimited https://github.com/search?q=path%3A.yarnrc&type=code

Reply:
Checking

Reply:
fixed.